src/integrationTest/java/com/hivemq/extensions/discovery/s3/S3DiscoveryFailIT.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2018-present HiveMQ GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hivemq.extensions.discovery.s3;
```

Comment:
```suggestion
package com.hivemq.extensions.cluster.discovery.s3;
```

Reply:
Is this supposed to be changed for all packages or just the test classes?